lochness/redcap/__init__.py

```diff
@@ -238,6 +238,10 @@ def get_run_sheets_for_datatypes(api_url, api_key,
 
             content_dict_list = json.loads(content)
 
+            # select the right form, and ignore the weird empty dictionary
+            content_dict_list = [x for x in content_dict_list if
```

Comment:
Note on this issue

Here are updates from what I observed. (Below, the data dictionary refers to the list of fields data included in { })

1. According to our design, we should have only two run sheet data for EEG and MRI. But, some subjects have more than two data dictionaries included in the run sheet data, when the REDCap API pulls data. For an example, one record has three data dictionaries included in its EEG run sheet form.

2. Updating run sheet data from the web interface for baseline and follow-up run sheets, only changes two data dictionaries. The extra data dictionary is not altered. But still being downloaded by REDCap API.

3. This extra data dictionary has '' in all fields, while the correct baseline and follow up data dictionary have `0`, `1`, or `2`.

I do not fully understand why REDCap is producing this empty data dictionary. To go forward with AMP-SCZ project, I'll update lochness to ignore this empty data dictionary included in the run sheet forms with '' in all fields.